packages/interface/src/contexts/Maci.tsx

```diff
@@ -122,6 +124,27 @@ export const MaciProvider: React.FC<MaciProviderProps> = ({ children }: MaciProv
         setSgData(attestationId);
         setIsLoading(false);
         break;
+      case GatekeeperTrait.Hats:
+        if (!signer) {
+          return;
+        }
+        getHatsSingleGatekeeperData({
+          maciAddress: config.maciAddress!,
+          signer,
+        }).then(({ criterionHat }) => {
+          // we need to check if we are allowed to signup
+          const hatsClient = getHatsClient();
+          hatsClient.isWearerOfHat({ wearer: address!, hatId: BigInt(criterionHat.at(0)!) }).then((res) => {
+            if (res) {
+              // we don't need any specific value here, as it will check based on the address
+              setSgData("0x0000000000000000000000000000000000000000000000000000000000000000");
```

Comment:
I am not sure what does this mean? (like why this specific data?

Reply:
this is just an empty bytes value (the one we have as default in maci cli) - as no sgGatekeeperData is needed for this call, it just needs to be a valid solidity bytes value